src/NuGet.Core/NuGet.ProjectManagement/INuGetProjectContext.cs

```diff
@@ -59,4 +61,20 @@ public enum MessageLevel
         /// </summary>
         Error
     }
+
+    /// <summary>
+    /// Enum for the type of NuGetProjectAction
+    /// </summary>
+    public enum NuGetProjectActionType
+    {
+        /// <summary>
+        /// Install
+        /// </summary>
+        Install,
```

Comment:
When action is Update, ActionType will be set to install, for this case, only delete packages.config for uninstall, so all others can be set to install.